packages/zosfiles/src/methods/copy/doc/ICrossLparCopyDatasetOptions.ts

```diff
@@ -44,11 +44,4 @@ export interface ICrossLparCopyDatasetOptions extends ICopyDatasetOptions {
      * @type {boolean}
      */
     overwrite?: boolean;
-
-    /**
-     * Prompt callback that will be invoked before overwiting a data set.
-     * @param targetDSN Name of data set that already exists
-     * @returns True if target data set should be overwritten
-     */
-    promptFn?: (targetDSN: string) => Promise<boolean>;
```

Comment:
smart! 
moving it to a more common interfaces will help to expose it in other places 🙏